framework/set/resources/rancher2/setProvidersAndUsersTF.go

```diff
@@ -72,7 +72,7 @@ func createRequiredProviders(rootBody *hclwrite.Body, configMap []map[string]any
 	terraformConfig := new(config.TerraformConfig)
 	operations.LoadObjectFromMap(config.TerraformConfigurationFileKey, configMap[0], terraformConfig)
 
-	source, rancherProviderVersion, awsProviderVersion, linodeProviderVersion, localProviderVersion, rkeProviderVersion := getRequiredProviderVersions(configMap)
+	source, rancherProviderVersion, cloudProviderVersion, linodeProviderVersion, localProviderVersion, rkeProviderVersion := getRequiredProviderVersions(configMap)
```

Comment:
This is very close. You will need to ensure that `linodeProviderVersion` is removed as well as you're consolidating to `CLOUD_PROVIDER_VERSION`. We'll talk offline in terms of how to test this to make sure nothing broke.